kubernetes-external-secrets/00-namespace.yaml

```diff
@@ -0,0 +1,4 @@
+apiVersion: v1
+kind: namespace
+metadata:
+  name: kubernetes-external-secrets
```

Comment:
We should use ensure-namespace for this and make sure there's a group setup too. Make cluster-admins members of the group